### DISCUSSION TITLE
Azure Filebeat Install

### DISCUSSION THREAD

**Discussion by georgehammock** [Security-Onion-Solutions/securityonion#10201] [Unsupported Versions]
Trying to ingest logs from my event hub on Azure to security onion. While going through the process, I did have a few questions. 

The first one is on top of the documentation found on installing file beat on the security onion side,  do we also need to go through the quick installation guide found here:
https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation-configuration.html

Or is configuring Filebeat on the file found /opt/so/saltstack/local/pillar/minions/ good enough? Documentation that I am using is here.   https://docs.securityonion.net/en/2.3/filebeat.html

**Comment by InfosecGoon** [comment]
Filebeat is already installed on your Security Onion server. To configure it, follow these directions:

- Filebeat modules in SO: https://docs.securityonion.net/en/2.3/filebeat.html#modules
- 
- Filebeat Azure module options: https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-azure.html
